linalg/hypre.cpp

```diff
@@ -1542,17 +1549,27 @@ HypreParMatrix *HypreParMatrix::ExtractSubmatrix(const Array<int> &indices,
       }
       CF_marker[indices[j]] = -1;
```

Comment:
Shouldn't this `for` loop be outside of the `#ifdef`?

Reply:
Yes! Sorry about that. This has been fixed in 796c4b2